inbm/dispatcher-agent/requirements.txt

```diff
@@ -11,4 +11,5 @@ urllib3==1.26.18
 certifi==2023.7.22
 pyinstaller==5.13.1
 setuptools==65.5.1
-future==0.18.3
\ No newline at end of file
+future==0.18.3
+python-debian==0.1.49
```

Comment:
The addition of `python-debian` to `requirements.txt` should be accompanied by a justification in the PR description, explaining why this new dependency is necessary. [medium]